### PR TITLE
Optional labels fe

### LINKED ISSUE #946: [blocks] optional labels for certain inputs
For most form elements, labels are required for accessibility reasons but for _outputs_ the case is less clear. They certainly need useful descriptions but since they are not interactive form elements, the accesibility considerations are more around readability.

We need  to figure out how/ where we can make certain labels optional without compromising end user usabilty or accessibility.

Continuation of #704.

### PR DESCRIPTION
**Pull request by pngwn**
Follow up to #1080 

Closes #946.

`demo/blocks_outputs` is a good one to check the label-less components but worth manually testing too.

The `show_label` option is respected for any component that has a label, not just forms.

### CODE REVIEW

File: ui/packages/app/src/components/Audio/Audio.svelte [hunk 1]
```diff
@@ -25,6 +26,8 @@
 	$: _value = normalise_file(value, root);
 
 	let dragging: boolean;
+
+	$: console.log($$props);
```
Comment (abidlabs) [inline ui/packages/app/src/components/Audio/Audio.svelte:30]:
```suggestion
```

File: ui/packages/app/src/components/File/File.svelte [hunk 2]
```diff
@@ -22,6 +23,8 @@
 	$: _value = normalise_file(value, root);
 
 	let dragging = false;
+
+	$: console.log($$props);
```
Comment (abidlabs) [inline ui/packages/app/src/components/File/File.svelte:27]:
```suggestion
```

**Comment by abidlabs** [comment]
This is great!

**Review by abidlabs — APPROVED** [review]

File: ui/packages/atoms/src/BlockLabel.svelte [hunk 3]
```diff
@@ -18,9 +19,12 @@
 	// if (!label && label !== "" && filetype) {
 	// 	label = type_labels[filetype as keyof typeof type_labels].text;
 	// }
+
+	$: console.log(show_label);
```
Comment (pngwn) [inline ui/packages/atoms/src/BlockLabel.svelte:23]:
```suggestion
```

**Pull request merged**